src/modules/blog/components/blogListPage/index.js

```diff
@@ -1,7 +1,60 @@
 import React, { Component } from "react";
+import BlogCard from "./../blogCard/index";
+import axios from "axios";
+import { Helmet } from "react-helmet";
+import Pagination from "../../../../globals/pagination";
+import styles from "./style.module.css";
 
 export default class BlogListPage extends Component {
+  state = {
+    blogs: [],
+    currentPage: 1,
+    pageSize: 3,
+  };
+
+  componentDidMount() {
+    axios
```

Comment:
Don't make the component to deal directly with the `back-end` and use `blog.service` to do that.